plugins/lua/buildingplan/planneroverlay.lua

```diff
@@ -207,40 +207,47 @@ ItemLine.ATTRS{
 }
 
 function ItemLine:init()
-    self.frame.h = 1
+    --self.frame.h = 2
     self.visible = function() return #get_cur_filters() >= self.idx end
     self:addviews{
         widgets.Label{
-            frame={t=0, l=0},
-            text='*',
-            auto_width=true,
+            frame={t=0, l=1},
+            text=string.char(26),
+            --auto_width=true,
             visible=self.is_selected_fn,
         },
+        widgets.Label{
+            frame={t=0, l=2},
+            text={
+                {text=self:callback('get_item_line_text')},
+                --{text='[filter]', pen=self:callback('get_f_pen')},
+                --{text='[x]', pen=self:callback('get_x_pen')},
```

Comment:
no need to leave leftover comments in. they'll always be available in the git history